tfjs-backend-webgpu/src/kernels/conv2d_mm_vec4_webgpu.ts

```diff
@@ -72,8 +73,8 @@ export class Conv2DMMVec4Program implements WebGPUProgram {
     }
 
     [this.fitA, this.fitB] = this.getShapeFit(elementsPerThread);
-    this.shaderKey =
-        `conv2DMMVec4_${this.activation}_${this.fitA}_${this.fitB}`;
+    this.shaderKey = `conv2DMMVec4_${this.activation}_${this.isVec4}_${
```

Comment:
`this.isVec4` is not needed. `conv2DMMVec4` already means it's always true.

Reply:
done